.env.example

```
@@ -1,32 +1,55 @@
 # LLM_PROVIDER=amazon-bedrock
 # # Credentials optional if running in AWS environment with a native credential provider
 # # AWS_ACCESS_KEY_ID=your-aws-access-key-id
 # # AWS_SECRET_ACCESS_KEY=your-aws-secret-access-key
 # AWS_REGION=us-east-1
 # BEDROCK_MODEL=global.anthropic.claude-sonnet-4-5-20250929-v1:0
+# BEDROCK_HIGH_CAPABILITY_MODEL=anthropic.claude-3-5-sonnet-20240620-v1:0
+# BEDROCK_MID_CAPABILITY_MODEL=anthropic.claude-3-sonnet-20240229-v1:0
+# BEDROCK_LOW_CAPABILITY_MODEL=anthropic.claude-3-haiku-20240307-v1:0
 # BEDROCK_TEMPERATURE=0.2
 
 
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)